src/single-movie-component/single-movie-component.js

```diff
@@ -22,26 +22,48 @@ class SingleMovieView extends Component {
     if (!this.state.isLoading) {
       return (
         <section className="single-movie-dashboard">
-          <img src={this.state.movie.backdrop_path} alt={`Backdrop of ${this.state.movie.title}`}/>
+          <span className="side-left"></span>
           <div className="this.state.movie-info">
-            <h2>Movie Title</h2>
-            <p>{this.state.movie.title}</p>
-            <h2>Summary</h2>
+            <img src={this.state.movie.backdrop_path} alt={`Backdrop of ${this.state.movie.title}`}/>
+            <h1>{this.state.movie.title}</h1>
             <p>{this.state.movie.overview}</p>
-            <h2>Rating</h2>
-            <p>{`${this.state.movie.average_rating.toFixed(2)} out of 10`}</p>
-            <h2>Genres</h2>
-            <p>{this.state.movie.genres.join(', ')}</p>
-            <h2>Release Date</h2>
-            <p>{this.state.movie.release_date}</p>
-            <h2>Movie Runtime</h2>
-            <p>{`${this.state.movie.runtime} minutes`}</p>
-            <h2>Budget</h2>
-            <p>{`$${this.state.movie.budget}`}</p>
-            <h2>Revenue</h2>
-            <p>{`$${this.state.movie.revenue}`}</p>
+            <div className="details-wrapper">
```

Comment:
The organization of this file looks so much better than before, very nice! Also the error handling for the revenue and budget are very clean.